packages/reactivity/src/reactive.ts

```diff
@@ -27,13 +27,17 @@ const readonlyValues = new WeakSet<any>()
 const nonReactiveValues = new WeakSet<any>()
 
 const collectionTypes = new Set<Function>([Set, Map, WeakMap, WeakSet])
-const observableValueRE = /^\[object (?:Object|Array|Map|Set|WeakMap|WeakSet)\]$/
+const observableTypes = new Set<String>(
+  ['Object', 'Array', 'Map', 'Set', 'WeakMap', 'WeakSet'].map(
```

Comment:
maybe you should `makeMap` instead
see https://github.com/vuejs/vue-next/pull/275#issuecomment-541782367

Reply:
good suggestion！

Reply:
> maybe you should use `makeMap` instead
> see [#275 (comment)](https://github.com/vuejs/vue-next/pull/275#issuecomment-541782367)

good suggestion！